src/device-registry/models/Site.js

```diff
@@ -431,7 +430,6 @@ siteSchema.statics = {
           "devices.tags": 0,
           "devices.description": 0,
           "devices.isUsedForCollocation": 0,
-          "devices.createdAt": 0,
```

Comment:
enabled the createdAt field when getting device details with the get Sites endpoint. cc: @ja-odur